spec/lib/models/revocable_spec.rb

```diff
@@ -33,4 +33,26 @@
       expect(subject).not_to be_revoked
     end
   end
+
+  describe :revoke_previous_refresh_token! do
+    subject { FactoryGirl.build(:access_token, previous_refresh_token: 'old_refresh_token') }
+    previous_token = FactoryGirl.build(:access_token, token: "previous_token")
+    after { subject.revoke_previous_refresh_token! }
+
+    it 'revokes the previous token if present and sets the attribute :previous_refresh_token to ""' do
+      expect(Doorkeeper::AccessToken).to receive(:by_refresh_token).with(subject.previous_refresh_token).and_return(previous_token)
```

Comment:
I'd rather have the `let`s inside of the `it` block (see https://robots.thoughtbot.com/lets-not), but I if you strongly prefer that alternative I'm fine with that.